drupal/etc/nginx/robots.conf

```diff
@@ -0,0 +1,41 @@
+# We want assets under the "not allowed" category to not have the X-Robots-Tag header.
+# Having these paths come first and setting the value to empty means they will not have the header.
+~*/core/(.*).css      "";
+~*/core/(.*).js       "";
+~*/core/(.*).gif      "";
+~*/core/(.*).jpg      "";
+~*/core/(.*).jpeg     "";
+~*/core/(.*).png      "";
+~*/core/(.*).svg      "";
+~*/profiles/(.*).css  "";
+~*/profiles/(.*).js   "";
+~*/profiles/(.*).gif  "";
+~*/profiles/(.*).jpg  "";
+~*/profiles/(.*).jpeg "";
+~*/profiles/(.*).png  "";
+~*/profiles/(.*).svg  "";
+# Not allowed
+~*/core/(.*)                    "noindex, nofollow";
+~*/profiles/(.*)                "noindex, nofollow";
+~*/README.txt                   "noindex, nofollow";
+~*/web.config                   "noindex, nofollow";
+~*/admin/(.*)                   "noindex, nofollow";
+~*/comment/reply/(.*)           "noindex, nofollow";
+~*/filter/tips/(.*)             "noindex, nofollow";
+~*/node/add/(.*)                "noindex, nofollow";
+~*/search/(.*)                  "noindex, nofollow";
+~*/user/register/(.*)           "noindex, nofollow";
```

Comment:
can we also add the header to `/user/register` path without a trailing slash?

- `/user/register`: don't have `x-robots-tag`
- `/user/register/`: have `x-robots-tag`